src/drt/src/io/io.cpp

```diff
@@ -1573,13 +1573,20 @@ void io::Parser::setCutLayerProperties(odb::dbTechLayer* layer,
             "Unsupported LEF58_SPACING rule for layer {} of type SAMEMASK.",
             layer->getName());
         break;
-      case odb::dbTechLayerCutSpacingRule::CutSpacingType::PARALLELOVERLAP:
-        logger_->warn(utl::DRT,
-                      260,
-                      "Unsupported LEF58_SPACING rule for layer {} of type "
-                      "PARALLELOVERLAP.",
-                      layer->getName());
+      case odb::dbTechLayerCutSpacingRule::CutSpacingType::PARALLELOVERLAP: {
+        auto con = make_unique<frLef58CutSpacingConstraint>();
+        con->setCutSpacing(rule->getCutSpacing());
+        con->setCenterToCenter(rule->isCenterToCenter());
+        con->setSameNet(rule->isSameNet());
+        con->setSameMetal(rule->isSameMetal());
+        con->setSameVia(rule->isSameVia());
+        con->setSameCut(rule->isSameCut());
+        con->setSameMask(rule->isSameMask());
+        con->setParallelOverlap(rule->isParallelOverlap());
```

Comment:
just set to true

Reply:
Do I only set ParallelOverlap to true? or all of the mentioned four rules?
Because I think at this point we are only sure that parallelOverlap is true.

Reply:
just paralleloverlap